cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -164,6 +166,7 @@ final class DataprocConf {
 
   private final boolean disableGCSCaching;
   private  final String troubleshootingDocsUrl;
+  private final boolean enableGCSCachingSnapshot;
```

Comment:
Same as above

Reply:
We may also add comments regarding the fact that this artifacts will be cached, but not removed. So only use for testing purposes.

Reply:
ack